packages/ra-ui-materialui/src/field/BooleanField.tsx

```diff
@@ -45,11 +46,10 @@ export const BooleanField: FC<BooleanFieldProps> = memo<BooleanFieldProps>(
         let ariaLabel = value ? valueLabelTrue : valueLabelFalse;
 
         if (!ariaLabel) {
-            ariaLabel =
-                value === false ? 'ra.boolean.false' : 'ra.boolean.true';
+            ariaLabel = !value ? 'ra.boolean.false' : 'ra.boolean.true';
```

Comment:
The test should be more precise and use `looseValue`, because otherwise I think it's a breaking change.